client/src/js/base/__tests__/ScrollList.test.js

```diff
@@ -1,227 +1,72 @@
-import { ScrollList, calculateScrollRatio } from "../ScrollList";
+import { ScrollList, calculateScrollRatio, getEventTarget } from "../ScrollList";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [getEventTarget not found in '../ScrollList'](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4770288)